src/timepicker/timepicker.ts

```diff
@@ -40,68 +40,70 @@ const NGB_TIMEPICKER_VALUE_ACCESSOR = {
     }
   `],
   template: `
-    <table>
-      <tr>
-        <td class="text-xs-center">
-          <button class="btn-link" (click)="changeHour(hourStep)"><span class="chevron"></span></button>
-        </td>
-        <td>&nbsp;</td>
-        <td class="text-xs-center">
-          <button class="btn-link" (click)="changeMinute(minuteStep)"><span class="chevron"></span></button>
-        </td>
-        <template [ngIf]="seconds">
-          <td>&nbsp;</td>
+    <fieldset [disabled]="disabled">
```

Comment:
While this is enough to disable all the controls we need to make sure that up / down chevrons are changing color when a button gets disabled. We need to have some visual clue to the users that a button is not clickable any more.